lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -106,9 +106,13 @@ SmallVector<unsigned> getSizePerThread(Attribute layout) {
   } else if (auto sliceLayout = layout.dyn_cast<SliceEncodingAttr>()) {
     return getSizePerThread(sliceLayout.getParent());
   } else if (auto mmaLayout = layout.dyn_cast<MmaEncodingAttr>()) {
-    assert(mmaLayout.getVersion() == 2 &&
-           "mmaLayout version = 1 is not implemented yet");
-    return SmallVector<unsigned>{2, 2};
+    if (mmaLayout.getVersion() == 2) {
+      return SmallVector<unsigned>{2, 2};
```

Comment:
Simply `return {2,2};` is fine, such way is suggested by clang-tidy.

Reply:
nice catch, thanks.